ui/app/components/clients/total-client-usage.js

```diff
@@ -93,52 +93,46 @@ export default class TotalClientUsage extends Component {
     let svgChartSize = chartSvg.node().getBBox(); // getBBox will return the width and height of svg element
     let yAxisScale = scaleLinear()
       .domain([0, max(dataset.map(d => d.total))]) // TODO will need to recalculate when you get the data
-      .range([svgChartSize.height + -TRANSLATE.down, 0]);
-
+      .range([svgChartSize.height, 0]);
     // Reference for tickFormat https://www.youtube.com/watch?v=c3MCROTNN8g
     let formatNumbers = number => format('.2s')(number).replace('G', 'B'); // for billions to replace G with B.
 
     // customize y-axis
     let yAxis = axisLeft(yAxisScale)
-      .ticks(6)
+      .ticks(7)
```

Comment:
on figma I see 6 ticks, was this to accomplish something else?

Reply:
changing the number didn't change the tick number...so this is still to-be-dealt with